ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -688,6 +692,10 @@ ethernet_interfaces:
     spanning_tree_bpduguard: < true | false >
     spanning_tree_portfast: < edge | network >
     vmtracer: < true | false >
+    priority_flow_control: < true | false >
+    priority_flow_control_priorities:
+      - priority : < 0-7 >
```

Comment:
Please update data model to make priority the key value, since it's unique.
```yaml
    priority_flow_control: < true | false >
    priority_flow_control_priorities:
      < 0-7 >:
        enable_pfc: < true | false >
```